app/src/main/java/org/fossasia/openevent/general/speakers/SpeakerViewHolder.kt

```diff
@@ -10,13 +10,14 @@ import kotlinx.android.synthetic.main.item_speaker.view.speakerOrgTv
 import org.fossasia.openevent.general.CircleTransform
 import org.fossasia.openevent.general.R
 import org.fossasia.openevent.general.utils.Utils
+import org.fossasia.openevent.general.utils.stripHtml
 
 class SpeakerViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
 
     fun bind(speaker: Speaker) {
         itemView.speakerNameTv.text = speaker.name
         itemView.speakerOrgTv.text = speaker.organisation
-        itemView.shortBioTv.text = speaker.shortBiography
+        itemView.shortBioTv.text = speaker.shortBiography?.stripHtml()
```

Comment:
Do this automatically in the class itself

Reply:
Okay

Reply:
done